### DISCUSSION TITLE
Writing a webserver

### DISCUSSION THREAD

**Discussion by wisehackermonkey** [lf-lang/lingua-franca#483] [General]
Im thinking of writing a webapp!
Any help would be apriated

**Comment by lhstrh** [comment]
I would recommend using the `TypeScript` target (see https://github.com/icyphy/lingua-franca/wiki/Writing-Reactors-in-TypeScript). You'll be able to use whatever npm module you prefer to handle incoming requests. A physical action can be used to trigger the creation of a  response. If you tell me what your favorite web application framework is, I'll show you some example Lingua Franca code that uses it.

**Comment by lhstrh** [comment]
Turns out we actually already had an example like this. Here it is: https://github.com/icyphy/lingua-franca/blob/master/example/TS/HTTPServerReactor/HTTPServer.lf
